ocs_ci/ocs/constants.py

```diff
@@ -103,6 +103,7 @@
 CEPHBLOCKPOOL_SC = "ocs-storagecluster-ceph-rbd"
 CEPHFILESYSTEM_SC = "ocs-storagecluster-cephfs"
 NOOBAA_SC = "openshift-storage.noobaa.io"
+LOCALSTORAGE_SC = "localblock"
```

Comment:
There is an existing constant at [#L1026](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L1026) which we can reuse